examples/rabbitmq-service-basic-auth/rabbitmq_service_basic_auth.md

```diff
@@ -1,6 +1,6 @@
 # RabbitMQ service - Basic authentication
 
-The underlying connection of the consumer service is secured with basic authentication.
+RabbitMQ connection to the server has an associated user which is authenticated. It also targets a virtual host for which the user must have a certain set of permissions. After an application connects to RabbitMQ and before it can perform operations, it must authenticate, that is, present and prove its identity. One of the primary ways of authenticating a client is by providing username/password credentials. In this example, the underlying connection of the listener is secured with basic authentication. A secured `rabbitmq:Listener` can be created by using the default host and port or custom configurations and providing the authentication details using the `rabbitmq:Credentials` record.
```

Comment:
```suggestion
RabbitMQ connection to the server has an associated user which is authenticated. It also targets a virtual host for which the user must have a certain set of permissions. After an application connects to RabbitMQ and before it can perform operations, it must authenticate, that is, present and prove its identity. One of the primary ways of authenticating a client is by using Basic Authentication. In this example, the underlying connection of the listener is secured providing username/password credentials. A secured `rabbitmq:Listener` can be created by using the default host and port or custom configurations and providing the authentication details using the `rabbitmq:Credentials` record.
```